newsApi.test.js

```diff
@@ -0,0 +1,149 @@
+const NewsApi = require('./newsApi');
+require('dotenv').config();
+
+require('jest-fetch-mock').enableMocks()
+
+describe('GuardianApi class', () => {
+  it('calls fetch and loads stories', async () => {
+    const api = new NewsApi();
+    fetch.mockResponseOnce(JSON.stringify(mockResponse));
+
+    api.getArticles((data) => {
+      expect(data.length).toBe(10);
```

Comment:
I'm still getting my head around mocking jest-fetch myself but the test looks good from what I can see! Not sure but does does it need to have all keys in the mock, like could 'isHosted, pillariD etc' be removed from the mock to make it shorter?

Reply:
Yeah very true. I just copied and pasted out the response using console.log, but you're right that there was unnecessary info in there